CONTRIBUTING.md

```diff
@@ -39,31 +39,25 @@ You will also need to install the Docker engine:
 Clone the repository:
 
 ```shell
-$ git clone https://github.com/hashicorp/consul-k8s.git
+git clone https://github.com/hashicorp/consul-k8s.git
```

Comment:
I think consul docs does start commandline commands with a `$`. I don't know if we have been consistent, but the desired pattern is to retain them.

Reply:
I talked about this with David a few months back. I think it would be best to move away from doing that because GitHub added a feature where code blocks have a button which will copy their contents. If we have the `$` in the code block then the user can't paste the command into their shell easily.

Reply:
Now, for the Consul docs site this is different because it has a feature which omits the `$` when you copy the command.

Reply:
oh cool! i didnt know that. The rich diff view does not show it but it looks like it does.